control-plane/subcommand/controller/command.go

```diff
@@ -329,6 +348,32 @@ func (c *Command) Run(args []string) int {
 	return 0
 }
 
+func (c *Command) configureCABundleUpdate() error {
```

Comment:
nit: maybe rename this function to `updateWebhookCABundle`? I think `configure` reads a bit counter-intuitive